components/nds-panel/__tests__/Panel.test.js

```diff
@@ -33,7 +33,7 @@ describe("Panel", () => {
 				Some panel content
 			</Panel>
 		);
-		expect(wrapper.props()["className"]).toEqual("panel panel--primary mt--0");
+		expect(wrapper.props()["className"]).toEqual("panel mt--0 panel--primary");
```

Comment:
Same issue as the other comment - modifier class should appear after the variant